ci_jobs_trigger/libs/operators_iib_trigger/iib_trigger.py

```diff
@@ -208,6 +206,16 @@ def get_iib_data_from_file(config_data):
         return {}
 
 
+def get_jobs_name_from_config(ci_jobs):
+    ci_jobs_name = []
+    for _, _jobs_data in ci_jobs.items():
+        if _jobs_data:
+            for _ci_job in _jobs_data:
+                ci_jobs_name.append(_ci_job["name"])
+
+    return ci_jobs_name
+
```

Comment:
[_ci_job["name"] for _ci_job in ci_jobs.keys()]

Reply:
@rnetser to get value of `name` key, we have to use the nested loop here

```suggestion
    jobs_data = [_job for _job_data in ci_jobs.values() if _job_data for _job in _job_data]
    return [job["name"] for job in jobs_data]
    
```

Reply:
@rnetser updated code and added condition to check if jobs exists, otherwise will get error: 
ERROR iib-trigger: Fail to run run_iib_update function. 'NoneType' object is not iterable

`[job["name"] for jobs in ci_jobs.values() if jobs for job in jobs]`